tech/database/redis/about.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Redis
+subsection: redis
+section: tech-database
+description: In-memory data structure store, used as a database, cache and message broker
+---
+
+# Redis
+
+Redis is a free and open source in memory data structure store. It is widely used as a database, in memory cache or a message broker.
+
+# Quick start
```

Comment:
Please use the Second level header  (`## Quick start`), the title should be only one.
```suggestion
## Quick start
```